west.yml

```diff
@@ -168,7 +168,7 @@ manifest:
       groups:
         - hal
     - name: hal_intel
-      revision: 89a0db5d9155484624447d62a5211bcf6f6f0387
+      revision: af547115d1c1e125f096a440f9583866a01c1bcf
```

Comment:
what is this sha? It Can't find it anywhere.

Reply:
it is from https://github.com/intel/hal-intel/pull/20.

Reply:
will update it after https://github.com/intel/hal-intel/pull/20 merged.